batch-models/src/main/scala/org/ekstep/analytics/dashboard/DashboardUtil.scala

```diff
@@ -170,7 +170,11 @@ case class DashboardConfig (
                              // for weekly claps
                              cutoffTime: Float,
                              // to enable/disable report sync
-                             reportSyncEnable: Boolean
+                             reportSyncEnable: Boolean,
+                             // warehouse report path
+                             warehouseReportPath: String,
```

Comment:
This variable is not added in the test util

Reply:
done